test/common/network/cidr_range_test.cc

```diff
@@ -323,14 +323,14 @@ TEST(Ipv6CidrRange, InstanceConstSharedPtrAndLengthCtor) {
 
 TEST(Ipv6CidrRange, StringAndLengthCtor) {
   CidrRange rng;
-  rng = CidrRange::create("::ffff", 122); // Assignment operator.
+  rng = CidrRange::create("ff::ffff", 122); // Assignment operator.
```

Comment:
Not off the top of my head. I don't think tests are dependent on the address value. Is the issue with OSX the leading ::?